
# For now, keep xine as default renderer and make gstreamer optional.
# This saves quite some disk space in dependency packages (e.g. for netbooks).

--- platform/gtk-x11/plat/options.py.orig	2009-07-24 15:20:37.000000000 +0200
+++ platform/gtk-x11/plat/options.py	2009-07-24 15:20:43.000000000 +0200
@@ -46,7 +46,7 @@
         self.help = help
 
 USE_RENDERER = GTKPref(key="useRenderer",
-                       default=u"gstreamer",
+                       default=u"xine",
                        alias="renderer",
                        help="Which renderer to use.  (gstreamer, xine)" )
 

